src/firefly/js/visualize/DrawLayerCntlr.js

```diff
@@ -287,27 +301,39 @@ export function dispatchDetachLayerFromPlot(id,plotId, detachPlotGroup=false,
 }
 
 /**
- * @summary Create plot layer containing the regions based on region file or region description
- * @param drawLayerId required
- * @param layerTitle  layerTitle is set based on drawLayerId or default setting it is unset
- * @param fileOnServer
- * @param regionAry
- * @param plotId The region layer is created on all plots of the active plot group in plotId is empty
- * @param dispatcher
+ * @summary Create drawing layer based on region file or region description
+ * @param {string} drawLayerId id of the drawing layer to be created, required
+ * @param {string} layerTitle  if it is empty, it will be created internally
+ * @param {string} fileOnServer region file name on server
+ * @param {string[]|string} regionAry  array or string of region description
+ * @param {string[]|string} plotId  array or string of plot id. If plotId is empty, all plots of the active group are applied.
+ * @param {function} dispatcher
+ * @param {Object} selectMode    redndering features for the selected region
```

Comment:
Looks like I can not omit selectMode parameter, I always have to specify it, even if I'd like to use the default settings. Unfortunately, eslint does not support the syntax below, which would be beneficial in this case:
{selectStyle = defaultRegionSelectStyle, selectColor = defaultRegionSelectColor, lineWidth = 0 }={}

Perhaps we can do the destructuring assignment in the method body?

We should also add the default values to jsdoc, ex. 
@param {string} [selectMode.selectStyle='UprightBox'] rendering style for the selected region ...